### DISCUSSION TITLE
EJBCA Username  / Password

### DISCUSSION THREAD

**Discussion by danieldownload** [grindsa/acme2certifier#193] [Q&A] [answered]
I'm facing the issue, that the username and password to issue a certificate against ejbca must be unique for every cert. Currently there is a fixed username and password defined in server.cfg, so it works exactly only for one cert. If I try to get another cert, ejbca tells me that the username already exists.

Any ideas to solve it?

**Comment by grindsa** [comment]
I remember the issue from long time ago and i believe it should be related to the entity configuration in ejbca. Unfortunately i cannot check my notes right now as i am travelling. Give me a few days, will keep you posted...

**Comment by danieldownload** [comment]
No problem, thx a lot.

Daniel

**Comment by grindsa** [comment, marked as answer]
Quick question after checking the configuration of my test-system where i do not have this issue: could it be that your entity profile limits the number of allowed requests?

<img width="936" alt="ee-profile-liit" src="https://github.com/user-attachments/assets/74e2ac0d-123e-407b-94c8-89ac4ad9e3e6">
